odevalidator/config.ini

```diff
@@ -2,16 +2,17 @@
 Path = metadata.bsmSource
```

Comment:
We should include the configuration file using setuptools. https://python-packaging.readthedocs.io/en/latest/non-code-files.html

Adding a `MANIFEST.in` file with:
```include odevalidator/config.ini```

That way the default configuration file is included in the package and third parties can download the package without having to set it as a git submodule.

Reply:
config.ini inclusion was added to setup.py. MANIFIEST.in did not work for some reason.